app/views/finance/invoices/unpaid.html.haml

```diff
@@ -0,0 +1,27 @@
+- title t('.title')
+
+- for supplier in @suppliers
+  - invoices = supplier.invoices.unpaid
+  - if invoices.any?
+    %h3= supplier.name
+    - invoices_sum = 0
+    - invoices_text = []
+    %p
+    - for invoice in invoices
+      - invoices_sum += invoice.amount
+      - invoices_text << invoice.number
+      = link_to finance_invoice_path(invoice) do
+        = format_date invoice.date
+        = ' ' + invoice.number
+      = ' ' + number_to_currency(invoice.amount)
+      %br/
+    %p
+      - if supplier.iban.present?
+        %b= heading_helper(Supplier, :iban) + ':'
```

Comment:
Sorry if it causes no problem, but doesn't the string addition undo any `html_safe` that may be present in the heading helper?

Reply:
the output is still safe html